kernel/memory/mapping.c

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: CGL-KFS
+// SPDX-License-Identifier: BSD-3-Clause
+
+/* kernel/memory/mapping.c
+ *
+ *
+ *
+ * created: 2022/12/12 - xlmod <glafond-@student.42.fr>
+ * updated: 2022/12/12 - xlmod <glafond-@student.42.fr>
+ */
+
+#include <stdint.h>
+
```

Comment:
Please add comments. What the function does? Why it's important? What are the flags?

Reply:
this file should not be here, i will remove it.